### PR TITLE
incentives test coverage

### PR DESCRIPTION
**Pull request by Psalmz777**
**_WHAT DOES THIS PR DO?_**

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/incentives/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Incentives API
+# Incentives aPI
```
Comment (Baalmart) [inline src/incentives/README.md:1]: @Psalmz777 was this small change intended? Seems to be relatively strange? :)

Comment (Psalmz777) [reply src/incentives/README.md]: Yes it was..I used that to check if the token was working ,i have changed it back

File: src/incentives/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "prod-pc": "SET NODE_ENV=production&&nodemon ./bin",
         "stage-pc": "SET NODE_ENV=staging&&nodemon ./bin",
         "prettier": "prettier --single-quote --print-width 80 --trailing-comma all --write 'src/**/*.js'",
-        "test": "mocha ./**/test/ut_*.js --exit"
+        "test": "nyc --reporter=lcov mocha ./**/test/ut_*.js --exit"
```
Comment (Baalmart) [inline src/incentives/package.json:16]: @Psalmz777 , I am still noticing different styles here. Care to shed more light here? Noticed another configuration where you introduced a new command of test-cov or something? That PR was for the data-mgt microservice and it has just been merged.

Comment (Psalmz777) [reply src/incentives/package.json:16]: @Martin it is essentially the same thing ,I worked on data-mgt first so I introduced a new script to cover for the test-coverage , while I was working on incentives though I noticed that I could append the test coverage in the same script as the test this the difference,as for the test cov on data-mgt , I hadn't worked on mocha tests before so it was one of the first methods I landed on but I noticed I could do it better that's why the test-cov isn't there in the final draft

**Comment by Baalmart** [comment]
Hi @Psalmz777 , what is the actual status of this PR? It has 2` labels` on it.

**Comment by Psalmz777** [comment]
> Hi @Psalmz777 , what is the actual status of this PR? It has 2` labels` on it.

Hi @Baalmart Its ready for review

**Review by Baalmart — APPROVED** [review]

**Pull request merged**